public/static/plugins/perf/js/trend_chart.js

```diff
@@ -593,16 +593,25 @@ var drawSingleTrendChart = function(params) {
     })
 
     focusedText
-      .attr('y', function(d, i) { return opsLabelsY[i] })
+      .attr({'y': function(d, i) { return opsLabelsY[i] },
+             'transform': function(){
```

Comment:
Two minor notes:
* quotes are not required
* Your approach is OK, but following is a bit more common:
```javascript
.attr({
  y:  function(d, i) { return opsLabelsY[i] },
  transform: function() {
    // ...
  }
})
```

Reply:
picked the wrong one to copy.